README.md

```diff
@@ -1 +1,50 @@
-# forks-and-clones-readme
+# Forks and Clones
+
+## Objectives
+
+1. Fork a repo
+2. Clone a repo
+3. Push to different remote repos
+
+
+##Forking a Github Repository
+Forking a Github repository is just a way to create your own copy of any other Github repo. We do this all the time while using Learn, so you can use your copy as a sandbox to play around while maintaining a canonical repo where the original lab remains intact. In other words, it creates another remote that allows you to work in your own world without having to mess with our learn-co-student stuff.
+
+You can fork any repo by clicking the "Fork" button at the top right any Github repository. 
+
+![](https://github-images.s3.amazonaws.com/enterprise/2.2/assets/images/help/repository/fork_button.jpg)
+
+[More on forking at the Github docs.](https://help.github.com/enterprise/2.2/user/articles/fork-a-repo/)
+
+##Cloning
+Now we want to get our forked copy of the repository on our local machine. This is where cloning is useful. To do this:
+
+1. Find your forked repo on you Github profile
```

Comment:
An easier one is click the fork button on the lab, then select your profile it'll take you to your fork.

Also mention that in the URL they can just change learn-co-students to their profile name